recon_surf/recon-surf.sh

```diff
@@ -286,24 +283,10 @@ case $key in
     fssurfreg=1
     shift # past argument
     ;;
-    --hires)
-    echo "WARNING: --hires is deprecated, use --vox_size auto."
-    vox_size=1
-    hiresflag="-hires"
+    --hires|--vox_size)
```

Comment:
since we are changing the interface for 2.0 release anyway, I think this can be removed and an error given if someone passes any unknown parameter option.